### PR TITLE
Fix close modal with escape keydown and cleanup import react

### PR DESCRIPTION
**Pull request by fouteox**
Hi, I fixed modal closing by adding "show" in useEffect dependency on DeleteUserForm page.

Then I cleaned up React calls that are no longer needed.

### CODE REVIEW

**Comment by fouteox** [comment]
add fix for label input

File: stubs/inertia-react/resources/js/Components/TextInput.jsx [hunk 1]
```diff
@@ -17,6 +17,7 @@ export default forwardRef(function TextInput(
             <input
                 type={type}
                 name={name}
+                id={name}
```
Comment (jessarcher) [inline stubs/inertia-react/resources/js/Components/TextInput.jsx:20]: I'm not a fan of re-using the `name` prop here as sometimes they need to be different.

I think it should be `id={id}` with a new `id` prop added to the component definition and everywhere the component is used with a label.

**Review by jessarcher — CHANGES_REQUESTED** [review]
Hey @fouteox,

Thanks for the PR. I've confirmed this fixes the close on escape and label behaviour, although I've left a comment regarding the latter.

As for the `React` import, I'm not super familiar with React and I appreciate any improvements that can be made.

I've tried your PR though, and I get the following warning during the SSR build: 

image

You can see it in CI [here](https://github.com/laravel/breeze/actions/runs/3542345661/jobs/5947734957#step:7:41) as well.

And when I log in to the application, nothing renders on the dashboard route and I get this error in the browser console:

image

I would like to get the other changes merged so it might be better to move the import changes to a separate PR if you'd still like to pursue it.

Thanks again!

File: stubs/inertia-react/resources/js/Components/TextInput.jsx (hunk 1 above)
Comment (fouteox) [reply stubs/inertia-react/resources/js/Components/TextInput.jsx]: I have add id in all component used TextInput.jsx

**Comment by fouteox** [comment]
Strange, I tried to compile on my side and I don't have this error, I sent a new commit by deleting the last import remaining in ssr.jsx, can you try again please?

**Comment by fouteox** [comment]
I finally found the problem with the React import, in the Dropdown file, so I fixed it.

I also refactored the Modal component using the power of headlessui which natively creates a portal and supports the keyboard.

**Review by jessarcher — APPROVED** [review]
Thanks, @fouteox! This looks and works great!

You've improved the functionality _and_ reduced the code :tada:

**Pull request merged**